Audit trail for monitors (#21294)
DataDog/documentation: content/en/monitors/manage/status.md

Events generated from your monitor (alerts, warnings, recoveries, etc.) are shown in this section based on the time selector above the **Status & History** section. The events are also displayed in your [Events Explorer][10].

### Audit trail

> Audit Trail automatically capture all monitor changes for all monitor types and create an event. This event explains the monitor changes (monitor edits for instance) showing the previous and the current configuration and displays the user that made the change. For more information, see the [Audit Trail][11] documentation and read the [Audit Trail best practices][12] blog.

Reviewer: ```suggestion
Audit Trail automatically captures monitor changes for all monitor types and creates an event. This event documents the changes to the monitor.

 For example, in the case of an edit to a monitor, the Audit Trail event shows:
 - The previous monitor configuration
 - The current monitor configuration
 - The user that made the change
 
 For more information, see the [Audit Trail][11] documentation and read the [Audit Trail best practices][12] blog.
```

[8]: /notebooks
[9]: /monitors/configuration/?tab=thresholdalert#evaluation-window
[10]: https://app.datadoghq.com/event/explorer
[11]: /account_management/audit_trail/

> [12]: /blog/audit-trail-best-practices/

Reviewer: ```suggestion
[12]: https://www.datadoghq.com/blog/audit-trail-best-practices/
```
One last fix.

Also, after I approve, should I merge this PR?

Author: Hi @urseberry,
we can merge it tomorrow when the change will be available in all the DC.

Thanks 🙇‍♂️
